package.json

```diff
@@ -42,7 +41,8 @@
   },
   "peerDependencies": {
     "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react-dom": "^0.14.0",
+    "react-tap-event-plugin": "^0.2.0"
```

Comment:
@oliviertassinari  Can you explain why this trigger that problem?  Didn't figure out it.  Thanks!